### DISCUSSION TITLE
Mapping with Leaflet

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#34] [Comments]
# Mapping with Leaflet

Father, husband, developer relations and web standards expert, and cat demo builder.

https://www.raymondcamden.com/2024/08/23/mapping-with-leaflet

**Comment by Skiranbase** [comment]
@Raymond Useful, thanks.

**Comment by WhiteStarLine** [comment]
Raymond, very helpful site and my thanks for what you do. Even the cat discussions!

I appreciate your thoughts on Leaflet.  I've used it almost every day since 2017 on a specialist military history GIS application that overlays historic trench maps / aerial photographs on satellite imagery plus Google StreetView,  Volunteers have georeferenced almost 8,000 maps and 50,000 Points of Interest into a web database.  A little bit of GeoJSON and spherical trigonometry and a user can click on a location and see what was there, what is there now and what is nearby.  The Austrian Alps trench maps give way to today's abandoned trenches and in some cases artillery pieces that were just too big to get down the mountain. Do-it-yourself vector overlays such as GeoJSON, KML or image overlay and manipulation are all possible.

My thanks to you for your efforts.  I discovered your blog post on table sorting when I was trying to present an HTML table of 8,000 maps to users with filtering, sorting and pagination. Worked perfectly.  I'm overdue to apply your recent post on JavaScript objects were the name of the key can be a variable.  Never realised that!

Keep up the good work!  Cheers, Bill

**Reply by cfjedimaster** [reply]
That is absolutely fascinating. Have you blogged or presented about this project?

**Comment by timGeo64** [comment]
Raymond: I concur with WhiteStarLine, Bill, a very helpful site and I am glad to see a post about GIS mapping. I am also impressed with how Bill is doing mapping.  I follow you, but I don't post much, but I had to now.

I am looking forward to you going into more detail with Leaflet and ways to integrate with ColdFusion.  I started web mapping quite a long time ago when Autodesk brought out MapGuide, good thing about that is they packaged it with, then, Macromedia ColdFusion. I think it was version 6 or 7 then.

Switched to GeoServer, Postgres GIS and Leaflet when MapGuide was discontinued.  I have been integrating maps into my web application for management of local governments since I started. I have been building in more functionality to the map as clients use it and keep asking for more. 

One example of integration with ColdFusion is selecting a parcel and getting the owners names list within a user specified distance and creating mailing letters using cfdocument.

I look forward to seeing what else you do with Leaflet.  Your site is extremely helpful to me, thanks.

- Tim

**Reply by cfjedimaster** [reply]
Given that I'm not doing much with CF now, is there something specific on the CF side that you think would be helpful? All I can imagine is perhaps loading in datapoints via an API call, but that would be an exercise in writing a CFC. :)
